src/archiefvernietigingscomponent/destruction/views.py

```diff
@@ -43,7 +47,12 @@
 # Views that route to the appriopriate specialized view
 
 
-class EnterView(LoginRequiredMixin, RedirectView):
+class EnterView(AccessMixin, RedirectView):
+    def dispatch(self, request, *args, **kwargs):
```

Comment:
I am not too sure how to go about this, because:

1. The `get_redirect_url()` method is reached only if the user is authenticated. So to reach the different TemplateView I think we would still need to overwrite the `dispatch()` method?
2. I think Joeri mentioned that he'd want the user to land first on the tutorial page (and not straight away be redirected to log in), as otherwise they might get confused as to what they need to do (especially in the demo-mode). 

So maybe a solution could be to use a `LoginRequiredMixin` and set the `login_url` to the start page, so that unauthenticated users are redirected there and there is no need to override the `dispatch()` method. It's maybe a bit of a hack though. :sweat_smile:

Reply:
Discussed with Joeri. The decision was to:
1. Make a new view for the startpage. Remove the demo banner from this page. This view should live in a different package than 'destruction'. Maybe a new 'core' package or in 'accounts'.
2. Make the entry view inherit again from the `LoginRequiredMixin` and set `login_url` to the url of the start page.